bentoml/_internal/bento/bento.py

```diff
@@ -245,7 +245,7 @@ def __init__(self, base_path: PathType):
 @attr.define(repr=False, frozen=True)
 class BentoInfo:
     tag: Tag
-    service: str = attr.field(converter=lambda svc: svc._import_str)  # type: ignore[reportPrivateUsage]
+    service: str  # type: ignore[reportPrivateUsage]
```

Comment:
rejected this. This should be fixed in #2049

Reply:
oh didn't see that one, i'll close the PR